jalejandromj/react-apprenticeship-capstone4
# Capstone-4_Deliverable-3

## Description
```
React router implemented por nav. Integration of Wizeline API for necessary views. Updated Home and ProductList pages. ProductDetail and SearchResult pages created.
```
## Type of Change

```
Delete options that are not relevant.
```


## How should this be tested?
```
Main files to review:
- Home.page; should:
Display correctly banners, product carousel (with button to add to cart), and categories. User should be able to navigate when 'onclick' on any product/category card. When 'onclick' a category, should redirect and automatically apply the respective filter.
- ProductDetail.page; should:
Display required product info. A static carousel should be displayed. A panel on the right side should let the user select an amount of eaches for that product to add to cart (dummy atm).
- ProductList.page; should:
Display all products from category selected and pagination controls enabled. User should be able to navigate when 'onclick' on any product. Filters enabled.
- SearchResult.page; should:
Redirect and display a list of products related to the user search. In order to search the user may use the input in the Header.
```
## Any background context you want to provide?
```
N/A
```
## What is the deliverable?
```
Capstone 4. Deliverable 3
```
## Screenshots and links
```
https://alejandromj-wizecommerce.netlify.app/
```
## Notes
```
App is responsive for 1280, 1366, and 1920 until now.
```
## Checklist
